File: CS/06_Annotations/SetFreeTextAnnotationSubject.cs
using Spire.Pdf;
using Spire.Pdf.Annotations;
using Spire.Pdf.Graphics;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SetFreeTextAnnotationSubject
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a new PDF document.
            PdfDocument doc = new PdfDocument();

            //Load the file from disk.
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\Template_Pdf_4.pdf");

            //Get the first page of PDF file.
            PdfPageBase page = doc.Pages[0];

            //Initialize a PdfFreeTextAnnotation.
            RectangleF rect = new RectangleF(150, 120, 150, 30);
            PdfFreeTextAnnotation textAnnotation = new PdfFreeTextAnnotation(rect);

            //Specify content.
            textAnnotation.Text = "\nSet free text annotation subject";

            //Set subject.
            textAnnotation.Subject = "SubjectTest";

            // Create a new PdfFont object using the Times Roman font family and a font size of 20
            PdfFont font = new PdfFont(PdfFontFamily.TimesRoman, 10);

            // Create a new PdfAnnotationBorder object with a specified width
            PdfAnnotationBorder border = new PdfAnnotationBorder(1f);

            // Assign the created font to the font property of the free text annotation
            textAnnotation.Font = font;

            // Assign the created border to the border property of the free text annotation
            textAnnotation.Border = border;

            // Set the border color of the free text annotation to gray
            textAnnotation.BorderColor = Color.Purple;

            // Set the line ending style of the free text annotation to slash
            textAnnotation.LineEndingStyle = PdfLineEndingStyle.Circle;

            // Set the background color of the free text annotation to light blue
            textAnnotation.Color = Color.Green;

            // Set the opacity of the free text annotation to 0.8
            textAnnotation.Opacity = 0.8f;
            page.Annotations.Add(textAnnotation);

            //Save the document
            String result = "SetFreeTextAnnotationSubject_out.pdf";
            doc.SaveToFile(result);


            //Launch the Pdf file
            PDFDocumentViewer(result);
        }

        private void PDFDocumentViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/06_Annotations/SetFreeTextAnnotationSubject.vb
Imports Spire.Pdf
Imports Spire.Pdf.Annotations
Imports Spire.Pdf.Graphics

Namespace SetFreeTextAnnotationSubject
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            ' Create a new PdfDocument object 
            Dim doc As New PdfDocument()

            ' Load an existing PDF document
            doc.LoadFromFile("..\..\..\..\..\..\Data\Template_Pdf_4.pdf")

            ' Get the first page of the PDF
            Dim page As PdfPageBase = doc.Pages(0)

            ' Create a rectangle for the annotation position and size
            Dim rect As New RectangleF(150, 120, 150, 30)

            ' Create a free text annotation with the specified rectangle
            Dim textAnnotation As New PdfFreeTextAnnotation(rect)

            ' Specify the content of the annotation
            textAnnotation.Text = vbLf & "Set free text annotation subject"

            ' Set the subject of the annotation
            textAnnotation.Subject = "SubjectTest"

            ' Set the font of the annotation
            Dim font As New PdfFont(PdfFontFamily.TimesRoman, 10)
            textAnnotation.Font = font

            ' Set the border of the annotation
            Dim border As New PdfAnnotationBorder(1.0F)
            textAnnotation.Border = border

            ' Set the border color of the annotation
            textAnnotation.BorderColor = Color.Purple

            ' Set the line ending style of the annotation
            textAnnotation.LineEndingStyle = PdfLineEndingStyle.Circle

            ' Set the color of the annotation
            textAnnotation.Color = Color.Green

            ' Set the opacity of the annotation
            textAnnotation.Opacity = 0.8F

            ' Add the annotation to the page
            page.Annotations.Add(textAnnotation)

            ' Specify the output file path for the modified PDF
            Dim result As String = "SetFreeTextAnnotationSubject_out.pdf"

            ' Save the modified PDF document to the output file
            doc.SaveToFile(result)

            ' Close the PDF document
            doc.Close()

            ' Launch the Pdf file
            PDFDocumentViewer(result)
        End Sub

        Private Sub PDFDocumentViewer(ByVal filename As String)
            Try
                Process.Start(filename)
            Catch
            End Try
        End Sub
    End Class
End Namespace
